frontend/src/components/forms/UserForm.vue

```diff
@@ -391,37 +410,31 @@ const choosenDistrictError = computed((): string => {
         <span class="font-bold">{{ chosenDistrictError[languagePref] }}</span>
       </p>
       <BaseSelect
-        v-if="userType === 'districtAdmin' && !userId"
+        v-else-if="userType === 'districtAdmin' || userType === 'newUser'"
         class="w-1/2"
         :label="langTranslations.userForm.districtSelectLabel"
         :options="[...districtMap.keys()]"
         v-model="chosenDistrict"
         :errorMessage="choosenDistrictError"
       />
       <BaseSelect
-        v-if="userType === 'districtAdmin' && !userId"
+        v-if="userType === 'districtAdmin' || userType === 'newUser'"
         class="w-1/2"
         :label="langTranslations.baseClubLabel"
         :options="[...clubMap.keys()]"
         v-model="chosenClub"
       />
       <!-- TODO: implications of changes to users role -->
       <BaseSelect
-        v-if="formType === 'siteAdminDistrict'"
+        v-if="formType === 'siteAdminDistrict' || formType === 'newAccount'"
         class="w-1/2"
         v-model="user.role_type"
         :label="langTranslations.roleLabel"
         :options="districtRole.filter((role) => role !== 'Webmaster')"
```

Comment:
ok so problem you only have distract roles showing , club roles should also be showing in the list  when a user is signing up .... so change the :options when in this mode to have all roles

Reply:
Gotcha, am I supposed to include the "Guest" option as well? It's part of clubRoles